recipes/terraclimate/meta.yml

```
@@ -0,0 +1,26 @@
+title: "TerraClimate Monthly Surface Dataset"
+description: "High-resolution monthly climate and climatic water balance dataset"
+pangeo_forge_version: "0.4.0"
```

```suggestion
pangeo_forge_version: "0.5.0"
```

This was just released and is needed for `subset_inputs` to work.